vietnamese/TansStory/mystory.md

```diff
@@ -0,0 +1,2 @@
+Minh di dinh cu o My cung duoc 10 nam roi. Minh moi hoc ra truong.
+Minh dang kiem [viec lam](https://raw.githubusercontent.com/thuynh8/create-your-own-adventure/master/vietnamese/TansStory/mystory.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.